### ISSUE TITLE
Examples: OpenGraph meta tags

### ISSUE THREAD

**Issue by Fyrestar** [mrdoob/three.js#24304]
This is a rather minor issue but could be fixed quickly i suppose, but if the example page is just static passing the tag to the iframe it could be fixed by outputting it through a php or other, just as wrapper for adding these tags based on the query.

![image](https://user-images.githubusercontent.com/28584767/177053425-6d0b927f-8ae0-491f-83e2-e876d4d886e7.png)

Some platforms like the forum also automatically create a link with the page title instead full link

![image](https://user-images.githubusercontent.com/28584767/177053853-e28fc393-fb66-437e-afc1-791d24bc220b.png)

It would be nice to provide the thumbnails we have since longer, for embedding in social media. And for the title using or appending the example name. The relevant og tags:

``` html
<meta property="og:title" content="" />
<meta property="og:type" content="website" />
<meta property="og:url" content="" />
<meta property="og:image" content="" />
<meta property="og:description" content="" />
```

**Comment by mrdoob** [comment]
Would the `og:image` be good enough?

**Comment by Fyrestar** [comment]
Yes, the title would ensure it grabs the name of the example though, and `og:type` one is just static/fixed.

**Comment by Mugen87** [comment]
I'm not sure I can follow. So the the title of a specific example, it's necessary to add it to `<meta property="og:title" content="" />`. AFAICS, this needs to be done when the site is served to the client. There is no chance to do this with GitHub pages.

**Comment by Fyrestar** [comment]
I see, yeah this was my concern with it being one static page and routing through the iframe, it would require any type of dynamic language like php to wrap the index with the meta tags, otherwise `og:image` for the thumbnails won't work either. The only thing that can be done then at most without hosting it on another platform is providing some generic image with couple thumbnails to make it more attractive.

Would have been more nice to get a thumbnail of the actual example, especially when you post multiple.

**Comment by mrdoob** [comment]
Oh, I thought you mean adding a an image like this in `examples/index.html`'s `og:image`.

![image](https://user-images.githubusercontent.com/97088/177690763-5abdf6c1-3e03-438c-a48f-25156b425412.png)

**Comment by Mugen87** [comment]
I remember that images use in `og:image` _should_ have a specific aspect ration and resolution. Does this still matter these days?

**Comment by Fyrestar** [comment]
Not really, almost every platform saves them on their servers for adapting it to their best suited resolution and size, i think with a collection of thumbnails like mrdoob posted it will be no issue anyway if some platform keeps rectangular and another crops quadratic for something being cut off. 16:9 should be the most usual.

**Comment by makc** [comment]
> this needs to be done when the site is served to the client. There is no chance to do this with GitHub pages.

_unless_ you do threejsexamples.org that will add them but still pull everything else from threejs.org / github
